program-runtime/src/prioritization_fee.rs

```diff
@@ -14,21 +17,32 @@ pub struct PrioritizationFeeDetails {
 
 impl PrioritizationFeeDetails {
     pub fn new(fee_type: PrioritizationFeeType, max_compute_units: u64) -> Self {
-        let mut compute_ticks = max_compute_units
-            .saturating_div(COMPUTE_UNIT_TICK_SIZE)
-            .max(1);
         match fee_type {
-            PrioritizationFeeType::Deprecated(fee) => Self {
-                fee,
-                priority: fee.saturating_div(compute_ticks),
-            },
+            PrioritizationFeeType::Deprecated(fee) => {
+                let priority = if max_compute_units == 0 {
```

Comment:
This got me thinking about the need for a minimum threshold that we can use to filter all of these from the buffer, i.e. return an error if less than this minimum

Reply:
Yeah, I was thinking about that as well. I think that de-prioritizing those transactions is probably ok for now. But seems useful to have a min on the request units instructions as well.

Reply:
> This got me thinking about the need for a minimum threshold that we can use to filter all of these from the buffer, i.e. return an error if less than this minimum

There might be some important lessons in EIP1559's github issue

"It is recommended that transactions with the same priority fee be sorted by time the transaction was received to protect the network from spamming attacks where the attacker throws a bunch of transactions into the pending pool in order to ensure that at least one lands in a favorable position."

Reply:
I don't exactly follow how that Ethereum issue translates to Solana, can you create a new issue and explain a bit more what you're suggesting?